src/services/timer.js

```diff
@@ -43,3 +43,30 @@ export const determineNextPeriod = (currentPeriod, counter) => {
 
   return PERIODS.work;
 };
+const notificatinFunc = (text, icon) => new Notification('Pomodoro', {
+  body: text,
+  icon,
+});
```

Comment:
Instead of naming this `notificatinFunc`, can you give it a more accurate name? Also, you have a spelling mistake here too. I suggest naming it `createNotification` or something like that.